project/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.4.4
```

Comment:
1.5.0?

Reply:
I'm not sure how this file got here, it's in `project/project`. I'll try deleting it.